src/intern/intern.js

```diff
@@ -0,0 +1,75 @@
+define({
+	proxyPort: 9000,
+
+	// A fully qualified URL to the Intern proxy
+    proxyUrl: 'http://localhost:9000/',
+
+    // Default desired capabilities for all environments. Individual capabilities can be overridden by any of the
+    // specified browser environments in the `environments` array below as well. See
+    // https://code.google.com/p/selenium/wiki/DesiredCapabilities for standard Selenium capabilities and
+    // https://saucelabs.com/docs/additional-config#desired-capabilities for Sauce Labs capabilities.
+    // Note that the `build` capability will be filled in with the current commit ID from the Travis CI environment
+    // automatically
+    capabilities: {
+        'browserstack.debug': false,
+        project: '<%- appName %>',
+        name: '<%- appName %>',
+        fixSessionCapabilities: false
+    },
+
+    // Browsers to run integration testing against. Note that version numbers must be strings if used with Sauce
+    // OnDemand. Options that will be permutated are browserName, version, platform, and platformVersion; any other
+    // capabilities options specified for an environment will be copied as-is
+    environments: [
+        { browserName: 'internet explorer', version: [ '10', '11' ], platform: 'WINDOWS' },
+        { browserName: 'firefox', platform: 'WINDOWS' },
+        { browserName: 'chrome', platform: 'WINDOWS' }/*,
+		 { browserName: 'Safari', version: '9', platform: 'OS X' }*/
```

Comment:
Maybe a few more examples should be commented out, e.g other browsers for Mac or Linux?